src/character.cpp

```diff
@@ -1491,6 +1497,14 @@ bool Character::can_run() const
     return get_stamina() > 0 && !has_effect( effect_winded ) && get_working_leg_count() >= 2;
 }
 
+bool Character::is_bp_armored( const bodypart_id &bp ) const
+{
+    return ( bp->token == bp_torso && has_bionic( bio_armor_torso ) ) || \
+           ( bp->token == bp_head && has_bionic( bio_armor_head ) ) || \
```

Comment:
Instead of using `bp->token == bp_head` use  `bp == body_part_head`. We're trying to get rid of enum-based bodyparts.

This also applies to the other places where `bp->token` is used.

Reply:
You appear to be reviewing an older version of these changes.

Reply:
Woops, my bad.